src/modules/dashboard/components/Dialogs/DAOIncorporationDialog/TabPanels/WhyPanel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Link from '~core/Link';
+import Tag from '~core/Tag';
+
+import styles from './TabPanels.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.DAOIncorporationDialog.TabPanels.WhyPanel.title',
+    defaultMessage: 'Why incorporate your DAO',
+  },
+  description: {
+    id: 'dashboard.DAOIncorporationDialog.TabPanels.WhyPanel.description',
+    defaultMessage: `Setup a legal wrapper for you DAO to extend it’s functionality and help protect contributors. Incorporation is done in Panama through a partner of Colony, Korporatio. Who is specifically focused on helping to support smart companies built and run on-chain.`,
```

Comment:
Please change to: "Setup a legal wrapper for you DAO to extend it’s functionality and help protect contributors. Incorporation is done in Panama through a service provided by Korporatio. Who is specifically focused on helping to support smart companies built and run on-chain."